compiler/src/iree/compiler/Pipelines/Options.h

```diff
@@ -95,6 +95,10 @@ struct GlobalOptimizationOptions {
   // Strips debug assertions after any useful information has been extracted.
   bool stripAssertions = false;
 
+  // Maximum byte size increase allowed for constant expr hoisting policy to
+  // allow hoisting.
+  int64_t constExprMaxSizeIncreaseThreshold = 2147483647;
```

Comment:
This should be 1024*1024 according to me. Also, do we need this if the pass default option already specifies this?